model/components/opencensus.yml

```diff
@@ -0,0 +1,31 @@
+---
+components:
+
+  # opencensus instrumentation toolkit
+  - id: opencensus
+    name: OpenCensus
+    description: A single distribution of libraries that automatically collects traces and metrics from your app, displays them locally, and sends them to any analysis tool.
+    link: https://opencensus.io/
+    logo: opencensus
+    license:
+      - Apache License 2.0
+    categories:
+      - instrumentation-lib
+    capabilities:
+      - tech:
+        - c++
+        - erlang
+        - php
+        - go
+        - java
+        - python
+        - ruby
+      - aspects:
+        - tracing
+        - metrics-app
+        - metrics-process
+    connections:
+        - prometheus-server
+        - stackdriver
+        - zipkin-server
```

Comment:
As far as I know, Stackdriver is not open-source. So let's omit it here.

Reply:
yes you are right it free to use but not OpenSource :)